### DISCUSSION TITLE
Keka Discussions

### DISCUSSION THREAD

**Discussion by aonez** [aonez/Keka#993] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DaddyJapan** [comment]
I have many projects going on and they overlap in timewise. I need to set a password for each customer and sent it to them once when the project starts. I also make a project folder for each customer. So if I go to a particular folder, and make a zip file for that customer, I  @would like to use a password for that customer. So I need to set a password for the folder before I create a zip file. If KEKA will provide me an interface between a folder and its password, then I can set a password for its folder once. After that I don't need to set a password for the files under that folder. I assume that I can reuse it until I change it.
I need to have an interface to specify a folder and a password for that folder. This should work for decompression too. When I move a zip file to that folder and unzip it from the folder. I assume that KEKA knows the password and unzip it.

Currently I need to set a password every time I create a zip file. It is cumbersome. Please consider for this scenario.

**Comment by carlsmoore** [comment]
Hi,   I'm a new user, and trying to better understand best steps to encrypt a file with a password, using a Macbook Pro.     Do I need to go into Keka and first set a password for the compression file type, and then compress the file?   Is there a way that if I compress the file that it could pop up and ask me for a password to set in a dialog box rather that setting a general password for a file type (zip, gzip etc)?

**Reply by aonez** [reply]
Right now the way to go with Keka is using the default password or opening Keka and setting a password each time you want to use that feature.

**Comment by carlsmoore** [comment]
@aonez Thanks for the quick response.    Understood.

**Comment by thisnugroho** [comment]
Hi, is it possible that when dragging a folder onto the app icon, the user can choose which files or subfolders to include in the archive? Currently, when I drag a folder, it gets compressed immediately. Just wondering if this option already exists?

**Reply by aonez** [reply]
There’s no such feature @thisnugroho. You can manually open the folder in the Finder, select those files and folder and drop them to the Keka icon/window or use the contextual menu options (if you enabled them).

**Reply by thisnugroho** [reply]
I understand,thank you for your response and hard work making this tool 🙏👍

**Comment by DreadNoughticus** [comment]
Is the dmg extraction based on the built in tools? or does it extract using a custom method. I have a very specific use case and am hoping this is the solution to my problems lol

**Reply by aonez** [reply]
Custom.
